### PR TITLE
feat(dynamicErrorPage): dynamic error page

### PR DESCRIPTION
**Pull request by ChanJit**
** Please provide as much detail as possible, but feel free to remove irrelevant sections **

need a different error message to display for expired job.  

BREAKING CHANGE:

one of the props in ErrorPage is been remove and been replace with different props.

`onRetryClick` => `button.onClick`

RFC URL:

none

MIGRATION GUIDE:

Before:

```
import {
  JobsDBErrorPage
} from 'seek-asia-style-guide/react';


<JobsDBErrorPage onRetryClick={function(){}} />
```

After:

```
import {
  JobsDBErrorPage
} from 'seek-asia-style-guide/react';


<JobsDBErrorPage
  customeButton={{
    onClick: () => {...}
  }}
 />
```

EXAMPLE USAGE:

```
import {
  JobsDBErrorPage
} from 'seek-asia-style-guide/react';


<JobsDBErrorPage
  customeButton={{
    onClick: () => {...},
    text: 'Search again'
  }}
  customeText={{
    headline: 'This job ad is currently unavailable',
    subline: 'Keep calm and search again'
  }}
/>
```

Screen Shot
-------------
error page

### CODE REVIEW

File: jobsDB/ErrorPage/ErrorPage.demo.js [hunk 1]
```diff
@@ -23,15 +23,84 @@ export default {
   initialProps: {},
   options: [
     {
-      label: 'onRetryClick',
-      type: 'checklist',
+      label: 'customeText',
```
Comment (ZhenYongSeekAsia) [inline jobsDB/ErrorPage/ErrorPage.demo.js]: customText?

Comment (ChanJit) [reply jobsDB/ErrorPage/ErrorPage.demo.js]: will update on it 👍

**Comment by MingHo138** [comment]
The preview from screenshot looks good to me. Will there be a testing environment of this style guide update?

**Comment by ChanJit** [comment]
sure @MingHo138 , you can preview it with following url http://seek-asia-style-guide--b87d0a85f9039c637923629ce7b1e55838d56af5.surge.sh/jobsdb-errorpage

File: jobsDB/ErrorPage/ErrorPage.js [hunk 2]
```diff
@@ -15,35 +15,38 @@ const ErrorSVG = () => {
   );
 };
 
-export default function JobsDBErrorPage({ onRetryClick }) {
+export default function JobsDBErrorPage({ customText, customButton }) {
```
Comment (ZhenYongSeekAsia) [inline jobsDB/ErrorPage/ErrorPage.js]: should be texts enough

**Pull request merged**